src/main/resources/l10n/Menu_de.properties

```diff
@@ -211,3 +211,5 @@ Disable_highlight_groups_matching_entries=Keine_Gruppen_hervorheben
 Online_help_forum=Online-Hilfeforum
 
 Manage_protected_terms=
+
+BibTeX=
```

Comment:
Maybe a blacklist would be a good idea. If the word is contained in the blacklist, then an error.